checkov/terraform/checks/resource/gcp/GoogleComputeFirewallUnrestrictedIngress3306.py

```diff
@@ -0,0 +1,17 @@
+from checkov.common.models.enums import CheckCategories
+from checkov.terraform.checks.resource.gcp.AbsGoogleComputeFirewallUnrestrictedIngress import AbsGoogleComputeFirewallUnrestrictedIngress
+
+# standard mysql port
+PORT = 3306
+
+
+class GoogleComputeFirewallUnrestrictedIngress3306(AbsGoogleComputeFirewallUnrestrictedIngress):
+    def __init__(self):
+        name = "Ensure Google compute firewall ingress does not allow unrestricted mysql access"
+        id = "CKV_GCP_87"
```

Comment:
```suggestion
        id = "CKV_GCP_88"
```
a check with ID 87 already exists, that's why the two tests failed